src/main/cc/wfa/virtual_people/core/model/utils/distributed_consistent_hashing.h

```diff
@@ -16,9 +16,18 @@
 #define WFA_VIRTUAL_PEOPLE_CORE_MODEL_UTILS_DISTRIBUTED_CONSISTENT_HASHING_H_
 
 #include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
 
 namespace wfa_virtual_people {
 
+struct DistributionChoice {
+  DistributionChoice(int32_t init_choice_id, double init_probability):
```

Comment:
Structs generally don't need a constructor. You can initialize with `DistributionChoice{some_choice_id, some_probability}`.

Reply:
Removed.